src/pedestal_toolbox/params.clj

```diff
@@ -68,6 +68,22 @@
             (assoc ctx :response (response/bad-request e)))))})
     {:schema schema}))
 
+(def query-param-accept
+  "A before interceptor that fakes an Accept header so that later
+  interceptors can handle the Accept header normally. This is used
+  because it's literally impossible to make a clickable link in a
+  browser that sets the Accept header in the normal way.
+
+  To use, add a accept=application/csv to the URL query string.
+  Expects the query params to have already been keywordized by
+  keywordize-params"
+  (interceptor
+   {:enter
+    (fn [ctx]
+      (if-let [content-type (get-in ctx [:request :params :accept])]
+        (assoc-in ctx [:request :headers "accept"] content-type)
```

Comment:
I'm getting nitpicky now, but this is *so* easy to confuse (I've done it a lot in the past), I'd rather see the `content-type` let var renamed `accept-type` or `accept` (or similar).